### PR TITLE
Fix aiofiles type definitions

### PR DESCRIPTION
**Pull request by moriyoshi**

### CODE REVIEW

**Comment by moriyoshi** [comment]
Can anyone in project take a look into this?

File: third_party/3/aiofiles/base.pyi [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 _T_co = TypeVar("_T_co", covariant=True)
 _V_co = TypeVar("_V_co", covariant=True)
 _T_contra = TypeVar("_T_contra", contravariant=True)
 
-class AsyncBase:
+class AsyncBase(Generic[_T]):
     def __init__(self, file: str, loop: Any, executor: Any) -> None: ...
-    async def __aiter__(self) -> Iterator[str]: ...
-    async def __anext__(self) -> str: ...
+    async def __aiter__(self) -> Iterator[_T]: ...
```
Comment (srittau) [inline third_party/3/aiofiles/base.pyi:11]: You could use this (`_C` needs to be defined):

```python
    async def __aiter__(self: Type[_C]) -> _C: ...
```

This way, the type returned by subclasses should be tighter. But please test this if you decide to go this way, as this is a fairly complex interaction.

**Review by srittau — APPROVED** [review]
Sorry for the late review. LGTM, but one optional suggestion below.

**Comment by achimnol** [comment]
I'm also encountering this issue and waiting for the fix to be deployed with mypy!

**Pull request merged**